engine/audio/src/audiocapture.cpp

```diff
@@ -151,10 +153,10 @@ double AudioCapture::fillBandsData(int number)
     // representing all the frequencies from 0 to m_sampleRate Hz.
```

Comment:
I believe this function is supposed to fill the data for only a single band numbered by `number`

Reply:
No, it seems to be computing all the bands for one whole bar graph simultaneously